code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -139,10 +139,32 @@
 	name = "tajaran leather gloves"
 	species_restricted = list(BODYTYPE_TAJARA)
 
-/obj/item/clothing/gloves/botanic_leather/vaurca 
+/obj/item/clothing/gloves/botanic_leather/vaurca
 	name = "vaurca leather gloves"
 	species_restricted = list(BODYTYPE_VAURCA)
 
+
+/obj/item/clothing/gloves/janitor
+	name = "rubber cleaning gloves"
+	desc = "A pair of thick, long rubber gloves, designed to prevent the wearer from the splash of industrial strength cleaners. Not electrically insulated."
```

Comment:
i get that they don't protect fully against electricity for game balance reasons, but rubber is an insulator. maybe add just a little bit of protection?

Reply:
Yeah fair